sway/sway-output.5.scd

```diff
@@ -190,6 +190,19 @@ must be separated by one space. For example:
 	may have no effect or produce unexpected output when used together with future
 	HDR support features.
 
+*output* <name> allow_tearing yes|no
+	Allows or disallows screen tearing as a result of asynchronous page flips,
```

Comment:
I'd avoid the term "asynchronous", because it's very ambiguous/overloaded. Maybe "immediate" would be better.

Reply:
Changed to "immediate".